src/icu/break_iterator.cr

```diff
@@ -24,20 +24,17 @@ class ICU::BreakIterator
     @ubrk = LibICU.ubrk_open(break_type, locale, nil, 0, pointerof(ustatus))
     ICU.check_error!(ustatus) { free }
 
-    ustatus = LibICU::UErrorCode::UZeroError
-    @utext = LibICU.utext_open_ut_f8(nil, @text.to_unsafe, @text.bytesize, pointerof(ustatus))
-    ICU.check_error!(ustatus) { free }
+    @chars = text.chars.map { |c| c.ord.to_u16 }
```

Comment:
Can you add a generic method/macro to do that somewhere, I think it may be useful in other classes ... (or maybe create/use an UChars class that wraps calls to `u_chars_to_u_chars` ?)